s/frontend/components/modals/views/report-user/report-user-modal-view.ts

```diff
@@ -14,13 +14,14 @@ export function reportUserModalView({
 	modals.popup({
 		closeOnBlanketClick: true,
 		renderPopup: ({close}) => html`
-			<div class="modalview report">
+		<namaka-banned-user></namaka-banned-user>
```

Comment:
let's revert changes to this file -- we don't actually want to put the "banned users" interface into the report-users modal view -- and this change is unrelated to board-stats anyways

Reply:
Fixed